adr/0008-code-owners.md

```diff
@@ -0,0 +1,100 @@
+# 8. Code owners
+
+Date: 2019-09-18
+
+Architecture issue: [#287](https://github.com/home-assistant/architecture/issues/287)
+
+## Status
+
+Draft
+
+## Context
+
+We get contributed a lot of new integrations, new features to integrations and refactors of integrations. The Home Assistant project is honored to receive so many great contributions to our project!
+
+Unfortunately, as a contributor, adding oneself as (the, or one of the) code owners of the integration contributed or contributed to, doesn't happen spontaneously that often.
```

Comment:
I don't think this is accurate for new integrations. Most contributors that add new integrations do add themselves. It's just that we have a very large majority of older integrations where we don't have a codeowner.

In this paragraph I'd describe that we have the codeowners item in manifest.json.

Reply:
Thanks, I'll rephrase that part. You are right.

I do want to avoid the mention of the `manifest.json`, since that is part of the "how", not the "why" or the decision itself.

Reply:
I've adjusted the working a bit to be more future proof and less based on history. Please let me know if this is acceptable for you @MartinHjelmare.

Reply:
Yes, thanks!